shims/pom.xml

```diff
@@ -85,6 +85,10 @@
             <id>release321</id>
             <modules><module>spark321</module></modules>
         </profile>
+        <profile>
+            <id>release301db</id>
+            <modules><module>spark301db</module></modules>
+        </profile>
```

Comment:
Why is this change here, and if it is really needed, why only 301db and not the other Databricks profiles?

Reply:
Added for all the spark releases. These are only used by idea, idea can add or remove submodule dynamically when switching the profiles in Maven panel.